beets/dbcore/types.py

```diff
@@ -207,6 +207,12 @@ class String(Type):
     sql = u'TEXT'
     query = query.SubstringQuery
 
+    def normalize(self, value):
+        if value is None:
+            return self.model_type()
```

Comment:
I think it would be cleaner to return `self.null` here (even though that is just `self.model_type()` for `String`.

Reply:
Yeah, you're absolutely right about that.